TYPO3.Flow/Classes/TYPO3/Flow/Persistence/PersistenceManagerInterface.php

```diff
@@ -27,13 +27,12 @@
      */
     public function injectSettings(array $settings);
 
-    /**
-     * Initializes the persistence manager, called by Flow.
-     *
-     * @return void
-     * @api
+    /*
+     * The "initialize" method has been deprecated and was already removed from the interface.
+     * It is still called until the next major Flow version, but you should replace it by
+     * using "initializeObject" instead.
+     * TODO: Remove for next major Flow version.
      */
```

Comment:
But it's gone from the class an replaced by initializeObject... See the change in 
TYPO3.Flow/Classes/TYPO3/Flow/Persistence/Doctrine/PersistenceManager.php